src/views/Team/Outlet.vue

```diff
@@ -2,97 +2,189 @@
   <ion-page>
     <ion-content>
       <div class="wrap">
-        <ion-spinner v-if="loading"/>
+        <div class="centralizeTotal" v-if="loading">
+          <ion-spinner />
+        </div>
         <template v-else>
           <div class="header" :style="teamStyle">
-            <div class="profile-img ion-padding">
-              <avatar size="10rem" :profile="team" />
-              <ion-chip v-if="canEdit" @click="selectNewAvatar()">
-                <ion-icon :icon="uploadIcon"></ion-icon>
-              </ion-chip>
-            </div>
-            <div class="ion-padding">
+            <ion-col size-md="2" size-lg="2" size-sm="2" size-xs="3">
+              <div class="profile-img">
+                <avatar size="7rem" :profile="team" />
+                <ion-chip v-if="canEdit" @click="selectNewAvatar()">
+                  <ion-icon :icon="uploadIcon"></ion-icon>
+                </ion-chip>
+              </div>
+            </ion-col>
+
+            <ion-col size-xl="8" size-md="8" size-sm="8" size-xs="7" offset="2">
               <h1 data-cy="title">
-                {{team.name}}
-                <ion-button data-cy-role="editModal" v-if="canEdit" color="light" @click="intendEditTitle" fill="clear">
-                  <ion-icon size="small" :icon="editIcon" />
-                </ion-button>
+                {{ team.name }}
+                <ion-icon
+                  size="small"
+                  :icon="editIcon"
+                  data-cy-role="editModal"
+                  v-if="canEdit"
+                  color="light"
+                  @click="intendEditTitle"
+                />
               </h1>
+
               <inline-link-list :items="socialLinks" showIcon>
-                <li v-if="canEdit">
-                  <ion-button @click="intendEditSocialLink" color="light" size="small" fill="clear">
-                    <ion-icon size="small" :icon="editIcon" />
-                  </ion-button>
-                </li>
+                <div v-if="canEdit" style="display: inline">
+                  <ion-icon
+                    size="small"
+                    :icon="editIcon"
+                    @click="intendEditSocialLink"
+                    color="light"
+                  />
+                </div>
               </inline-link-list>
+
+              <div class="extra-actions" v-if="canEdit">
+                <ion-chip
+                  title="remove background"
```

Comment:
while on it, let's i18n these:
```suggestion
                  :title="$t('team.edit.actions.remove_background')"
```

and alike...